web/toolbar.js

```diff
@@ -112,6 +112,17 @@ class Toolbar {
               : AnnotationEditorType.STAMP;
           },
         },
+        telemetry: () => {
+          eventBus.dispatch("reporttelemetry", {
+            source: this,
+            details: {
+              type: "editing",
+              data: {
+                action: "pdfjs.image.icon_click",
+              },
```

Comment:
Nit: This fits on one line.
```suggestion
              data: { action: "pdfjs.image.icon_click" },
```